fe/fe-core/src/main/java/com/starrocks/sql/optimizer/statistics/StatisticsCalculator.java

```diff
@@ -354,6 +355,13 @@ private HiveColumnStats computeHiveColumnStatistics(ColumnRefOperator column, Hi
 
     private Statistics.Builder estimateScanColumns(Table table, Map<ColumnRefOperator, Column> colRefToColumnMetaMap) {
         Statistics.Builder builder = Statistics.builder();
+        if (StatisticUtils.statisticTableBlackListCheck(table.getId())) {
```

Comment:
This check is already in CachedStatisticStorage::getColumnStatistics, is this necessary?

Reply:
delete